docs/commands/build-docs.md

```diff
@@ -0,0 +1,73 @@
+# `build-docs`
+
+## Introduction
+
+The `build-docs` command builds Redoc into a zero-dependency HTML file.
+
+## Usage
+
+```bash
+redocly build-docs <api>
+redocly build-docs <api> --output=custom.html
+redocly build-docs <api> --features.openapi.theme.colors.primary.main=orange
+redocly build-docs <api> -t custom.hbs
+redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page meta description"
+```
+
+
+## Options
+
+Option | Type | Description
+-- | -- | --
+api | string | Path to the API definition filename or alias that you want to generate the preview for. Refer to [the api section](#api) for more options.
+--output | string | Output file. Default value is `redoc-static.html`.
+--title | string | Page title.
+--disableGoogleFont | boolean | Disable Google Font. Default value is `false`.
+--cdn | boolean | Use latest Redoc version. Default value is `false`.
```

Comment:
I'm not sure I understand this. What version is used otherwise?

Reply:
If we run the `build-docs` command without `cdn` option we use the Redoc version from `package.json` inside `@redocly/cli` package.
Respectively, if we pass `cdn` option we'll use the latest version of Redoc.

This option we have in `redoc-cli` and migrated here for backward compatibility. Many users used this option in `redoc-cli`.

Reply:
```suggestion
--cdn | boolean | Uses the CDN pointing to the latest version of Redoc. If not enabled, the Redoc version is selected from Redocly CLI's associated dependency (and to update Redoc requires updating the CLI and building the docs again). Default value is `false`.
```